### PR TITLE
Hello Project

### PR DESCRIPTION
**Pull request by RodChak**
- Created a new branch according to the [Github flow] rules.
- Added a .gitignore file.
- Set up a linter for HTML and CSS, also making sure that any files that are not relevant are ignored by git.
- Added descriptive README file based on professional requirements.
- Added a HTML file that includes a header with the text "Hello Microverse".
- Added CSS file that includes one class with styles for the header.
- Added test.md file with the text "This file should be ignored by git".
- Made sure that the last file is actually ignored by git and is not present in my GitHub Repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,97 +1,37 @@
...
-
-If you completed the Sass lessons you may prefer to work with SCSS files, if that's the case you need to perform
-a few modifications to the linter configuration files.
-
-1. In the file `.github/workflows/linters.yml` [line 48](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml#L48) replace `"**/*.{css,scss}"` with `"**/*.scss"`
-2. Use `npx stylelint "**/*.scss"` to lint you SCSS files and not the generated CSS
-
-Once you've done the previous steps you can focus on your SCSS files and not in fixing the generated CSS output
-every time.
+This project is [MIT](./MIT.md) licensed.
```
Comment (tresorsawasawa) [inline README.md]: - [x] Kindly make sure you fix this broken link in the **License section** of the README file, **Line 37** (`[MIT](./MIT.md)`) to make it more professional. I think you can fix this by only adding this [MIT.md](https://github.com/microverseinc/readme-template/blob/master/MIT.md) license file in your root directory and edit it according to your information.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+header h1 {
+  color: #000;
+}
```
Comment (tresorsawasawa) [inline style.css]: - [x] Kindly make sure the `style.css` file includes one class with styles for your header [as required for the project](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=A%20.css%20file%20that%20includes%20one%20class%20with%20styles%20for%20your%20header%20(make%20it%20in%20your%20favorite%20color).). I suggest you add one class in the `<h1>` tag(in the index.html file) and then use it to style your header.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
```
Comment (tresorsawasawa) [inline index.html]: - [x] Kindly make sure to add the link of your styling file in the `index.html` file to make the styling work, as for now, no styling is working on your HTML code.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+<body>
+    <header>
+        <h1>Hello Microverse!</h1>
```
Comment (tresorsawasawa) [inline index.html]: - [ **OPTIONAL** ] **IMPORTANT:** I suggest you use a different color from `#000 or Black` so you can check if your styling is working once you have added the styling link in the `index.html` file(using  `#000 or Black` color will not let you know if your styling is working or not as it is the default used color in HTML). I know you can do it :muscle: :muscle:

File: README.md [hunk 5]
```diff
@@ -1,97 +1,37 @@
...
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-If you are not familiar with linters, read [root level README](../README.md).
+# Hello Microverse Project
 
-## Set-up GitHub Actions
+In this project, I set up a "Hello world" repository. No complex coding was used for this exercise. My goal here is to master all of the tools and best practices I learned about in previous steps.
 
-Please do the following **steps in this order**:
+## Getting Started
```
Comment (tresorsawasawa) [inline README.md:8]: - [ ] Kindly make sure you also complete the **## Getting Started** section as it is empty to make your **Readme** documentation more professional like adding some information about **cloning the repo locally for someone who needs its copy**, how to set up the app, install dependencies,... I suggest you take a look at [this template](https://github.com/tresorsawasawa/MyPortfolio/blob/develop/README.md) and use it as an example

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @RodChak 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [x] **Kindly make sure your Pull Request has a descriptive title telling what changes are inside. i.e you should update the current Pull Request title to a descriptive one.**
- [x] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (danmainah) [inline README.md]: - [x] @RodChak as previously requested kindly add `MIT.md` file or remove the license altogether from your readme file.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @RodChak, 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Github flow was correctly used


 # Required Changes ♻️

- [x] Kindly update your pull request title to be descriptive as previously requested eg you could add `Hello Project` .
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
Hi @RodChak, 

Good implementation of the changes required, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- [ ] Good implementation of linters :heavy_check_mark: 
- [ ] Good descriptive Pull Request title :heavy_check_mark: 
- [ ] Good commit messages :heavy_check_mark: 

## Optional suggestions

- [ OPTIONAL ] As mentioned by the previous reviewers, you can add the [MIT.md](https://github.com/microverseinc/readme-template/blob/master/MIT.md) file to your root directory, is not mandatory but it can allow your project to look more professional:sunglasses: :sunglasses:, if you don't need it also kindly delete it in your `README.md` file.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (RodChak) [reply README.md]: Hey @danmainah, thank you for the feedback, really appreciate it. Could you please assist me with this part, I've already made 3 changes thinking I'm doing the correct thing, but it seems I'm not doing it correctly. I've added this [MIT.md](https://github.com/microverseinc/readme-template/blob/master/MIT.md) license file in my root directory, but my third reviewer still insisted that I correct that part, could you please assist me if you have some time, I've attached a screenshot of how I did it in my README.md file.
Screenshot 2022-06-29 1

**Pull request merged**